.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           name: Run tests and linting
           command: |
             cp .env.sample .env
-            make check
+            make check_legacy
```

Comment:
Oh right....  Actually....  Instead of doing it this way, let's just take out `check` from the makefile and rename `check_legacy` to `check` so that way running `make check` locally is the same as in CI and we don't have to worry about remembering which one to use.